### ISSUE TITLE
remove white space before <notifier-container></notifier-container>

### ISSUE THREAD

**Issue by lidroider** [dominique-mueller/angular-notifier#113]
Version 4.1.1, angular 6
I can't remove the white space before <notifier-container></notifier-container>
If I set `display: hidden` in class `notifier__container` then notify don't display.
If I set `display: block; height: 0px` in class `notifier__container` then the white space still be there

![image](https://user-images.githubusercontent.com/12046727/48457573-d157dc00-e7f5-11e8-871c-63e3eafaf5df.png)
In the image: red is `<notifier-container></notifier-container>`, blue is my main component, I want to Remove the white space
Please help me
Thanks

**Comment by sandeepsuvit** [comment]
This can be fixed by wrapping the component in a div and marking the height as 0.
1. In the app.component.html

``` html
<div class="notifier">
    <notifier-container></notifier-container>
</div>
```

2. app.component.css
```css
.notifier {
    height: 0;
    overflow: hidden;
}
```

**Comment by dominique-mueller** [comment]
I'm not able to see this issue in the demo application. Perhaps it's due to custom styles you have in your application? Any chance you can create a reproduction of this issue?

**Comment by dominique-mueller** [comment]
Released with **[9.0.1](https://github.com/dominique-mueller/angular-notifier/releases/tag/9.0.1)**.
